src/middleware/resolveTemplate.js

```diff
@@ -11,7 +11,9 @@ const viewTargets = (step, viewFolders) => {
   return viewFolders
     .map(folder => [
       path.resolve(path.join(folder, `${step.name}.html`)),
-      path.resolve(path.join(folder, `${step.name}.template.html`))
+      path.resolve(path.join(folder, `${step.name}.template.html`)),
+      path.resolve(path.join(folder, `${step.name.toLowerCase()}.html`)),
+      path.resolve(path.join(folder, `${step.name.toLowerCase()}.template.html`))
```

Comment:
This seems very specific to your implementation.

Reply:
i did try changing files to use uppercase, but when i logged them out on CNP it couldnt find them....